tests/rados/test_scrub_flag_pool_level.py

```diff
@@ -0,0 +1,361 @@
+        assert (
+            "POOL_SCRUB_FLAGS: Some pool(s) have the noscrub, nodeep-scrub flag(s) set"
+            in health_detail
+        )
+        log.info(
+            "POOL_SCRUB_FLAGS: Some pool(s) have the noscrub, nodeep-scrub flag(s) set message appeared"
+        )
+        log.info(f"Health warning: \n {health_detail}")
+        log.info(
+            "END-Verification of the noscrub and nodeep-scrub flags against the %s pool",
+            inactive_test_pool,
+        )
+        # Get the scrub and dep-scrub timings
+        set_scheduled_scrub_parameters(scrub_object)
+
+        is_scrub_complete = False
+        end_time = datetime.now() + timedelta(seconds=1200)
+        while datetime.now() <= end_time:
+            active_pool_after_scrub_stamps = get_scrub_stamps(
+                rados_object, active_test_pool
+            )
+            log.info(
+                "The time stamps after testing on the active pool - %s is %s",
+                active_test_pool,
+                active_pool_after_scrub_stamps,
+            )
+            scrub_result = compare_pg_scrub(
+                active_pool_before_scrub_stamps, active_pool_after_scrub_stamps
+            )
+            if scrub_result:
+                is_scrub_complete = True
+                break
+            log.info("Scrub operations are in progress on %s pool", active_test_pool)
+            log.info("Waiting 15 seconds before retrying...")
+            time.sleep(15)
+
+        if not is_scrub_complete:
+            log.error("The scrub operations are not completed")
+            return 1
+
+        inactive_pool_after_scrub_stamps = get_scrub_stamps(
+            rados_object, inactive_test_pool
+        )
+        log.info(
+            "The time stamps before testing on the active pool - %s is %s",
+            inactive_test_pool,
+            inactive_pool_after_scrub_stamps,
+        )
+        scrub_result = compare_pg_scrub(
+            inactive_pool_before_scrub_stamps, inactive_pool_after_scrub_stamps
+        )
+        if scrub_result:
+            log.error(
+                "The scrub operations are started on the %s pool which the noscrub and nodeep-scrub "
+                "flags are set",
+                inactive_test_pool,
+            )
+            return 1
+
+        # Set the noscrub and nodeep-scrub flags to the pool name
+        rados_object.set_pool_property(
+            pool=active_test_pool, props="noscrub", value="true"
+        )
+        rados_object.set_pool_property(
+            pool=active_test_pool, props="nodeep-scrub", value="true"
+        )
+
+        # Perform the user initiated scrub
+        rados_object.run_scrub(pool=active_test_pool)
+        rados_object.run_scrub(pool=inactive_test_pool)
+        rados_object.run_deep_scrub(pool=active_test_pool)
+        rados_object.run_deep_scrub(pool=inactive_test_pool)
+
+        active_pool_before_scrub_stamps = get_scrub_stamps(
```

Comment:
Collect time-stamps for active/inactive pool before triggering the user scrubs

Reply:
Done.